community/kernel/src/test/java/org/neo4j/graphdb/IndexingAcceptanceTest.java

```diff
@@ -513,17 +513,17 @@ public void shouldIncludeNodesCreatedInSameTxInIndexSeekByPrefix()
         GraphDatabaseService db = dbRule.getGraphDatabaseService();
         IndexDefinition index = createIndex( db, LABEL1, "name" );
         createNodes( db, LABEL1, "name", "Mattias", "Mats" );
-        PrimitiveLongSet expected = createNodes( db, LABEL1, "name", "Karl", "Karlsson" );
+        PrimitiveLongSet expected = createNodes( db, LABEL1, "name", "Carl", "Carlsson" );
         // WHEN
         PrimitiveLongSet found = Primitive.longSet();
         try ( Transaction tx = db.beginTx() )
         {
-            expected.add( createNode( db, map( "name", "Karlchen" ), LABEL1 ).getId() );
-            createNode( db, map( "name", "Carla" ), LABEL1 );
+            expected.add( createNode( db, map( "name", "Carlchen" ), LABEL1 ).getId() );
+            createNode( db, map( "name", "Karla" ), LABEL1 );
             Statement statement = getStatement( (GraphDatabaseAPI) db );
             ReadOperations readOperations = statement.readOperations();
             IndexDescriptor descriptor = indexDescriptor( readOperations, index );
-            found.addAll( readOperations.nodesGetFromIndexRangeSeekByPrefix( descriptor, "Karl" ) );
+            found.addAll( readOperations.nodesGetFromIndexRangeSeekByPrefix( descriptor, "Carl" ) );
```

Comment:
@thobe There's no technical reason behind this. I was working on this particular test for a bit and since my nephew is named Carl with a C i felt the need to change this test accordingly.

btw, nice that this is getting worked in after all :+1: